docs/guides/research/research.rst

```diff
@@ -0,0 +1,39 @@
+Research
+---------
```

Comment:
I don't see a link to this document from the main documentation site after it's generated.

But the more important question is: do we want to have this? The HotOS paper should be the main source of information for the _research output_. It's, of course, fine to have architecture and design documents, but they should be addition to the paper.

Reply:
Oh, I can have a look. I think index.rst is missing. I can't read large papers, feels like a task. Feel free letting me know if you don't like, I can remove.